packages/cli/src/user-management/email/user-management-mailer.ts

```diff
@@ -50,30 +52,22 @@ export class UserManagementMailer {
 		if (!this.mailer) return { emailSent: false };
 
 		const template = await this.getTemplate('invite');
-		const result = await this.mailer.sendMail({
+		return await this.mailer.sendMail({
 			emailRecipients: inviteEmailData.email,
 			subject: 'You have been invited to n8n',
-			body: template(inviteEmailData),
+			body: template({ ...inviteEmailData, ...this.getBasePayload() }),
 		});
-
-		// If mailer does not exist it means mail has been disabled.
-		// No error, just say no email was sent.
-		return result ?? { emailSent: false };
```

Comment:
Does the calling code handle the email not being sent without this flag?

Reply:
this was dead code, since it was added over 2 years ago. `.sendMail` either sends the email, or it throws an error. there is no scenario where it will return a falsey value.